include/TS3Data.h

```diff
@@ -21,11 +23,16 @@ class TS3Data : public TGRSIDetectorData  {
     std::vector<UShort_t>  fS3_SectorDetector;
     std::vector<UShort_t>  fS3_SectorNumber;
     std::vector<TFragment> fS3_SectorFragment;
+    
+   static bool fIsSet;      //!
```

Comment:
My bad. You are right. This is in the data so all of this is irrelevant. For some reason I thought this was the hit. Sorry.

> On Oct 29, 2015, at 4:43 PM, jsmallcombe <notifications@github.com> wrote:
> 
> In include/TS3Data.h <https://github.com/GRIFFINCollaboration/GRSISort/pull/560#discussion_r43453909>:
> 
> > @@ -21,11 +23,16 @@ class TS3Data : public TGRSIDetectorData  {
> >      std::vector<UShort_t>  fS3_SectorDetector;
> >      std::vector<UShort_t>  fS3_SectorNumber;
> >      std::vector<TFragment> fS3_SectorFragment;
> > +    
> > +   static bool fIsSet;      //!
> I'm happy to follow the efficient style ... but this is the same in griffindata as s3data
> "bool TGriffinData::fIsSet = false;"
> 
> Could you highlight where you mean static variables? Its possible I copied something wrong when trying to match other detectors.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/GRIFFINCollaboration/GRSISort/pull/560/files#r43453909>.
>